cmd/admin.go

```diff
@@ -145,3 +154,41 @@ func runCreateUser(c *cli.Context) error {
 	fmt.Printf("New user '%s' has been successfully created!\n", c.String("name"))
 	return nil
 }
+
+func runRepoSyncReleases(c *cli.Context) error {
+
+	setting.NewContext()
+	models.LoadConfigs()
+
+	setting.NewXORMLogService(false)
+	if err := models.SetEngine(); err != nil {
+		return fmt.Errorf("models.SetEngine: %v", err)
+	}
+
+	repos, count, err := models.SearchRepositoryByName(&models.SearchRepoOptions{})
```

Comment:
2 things:
1. Should we use paging instead of dragging all repositories into memory at once?
2. We need set `Private` to `true`

See [models/repo_indexer.go](https://github.com/go-gitea/gitea/blob/f5155b99136b6e19ab494a97adfcee1810a3d5e7/models/repo_indexer.go#L82) for an example.